WordPress/src/main/res/layout/new_account_user_fragment_screen.xml

```diff
@@ -87,6 +87,7 @@
                     android:layout_height="wrap_content"
                     style="@style/WordPress.NUXEditText"
                     android:inputType="textUri"
+                    android:digits="0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
Did you try with `android:capitalize="none"` and `android:digits="0123456789abcdefghijklmnopqrstuvwxyz"` without using the listener (`afterTextChanged`) that replaces Capital letters?